packages/medusa/src/services/pricing.ts

```diff
@@ -450,62 +450,27 @@ class PricingService extends TransactionBaseService {
     return productPricing.get(productId)!
   }
 
-  /**
-   * @deprecated
-   * @param variants
-   * @param context
-   */
-  async setVariantPrices(
-    variants: ProductVariant[],
-    context?: PriceSelectionContext
-  ): Promise<PricedVariant[]>
-
-  /**
-   * Set additional prices on a list of product variants.
-   * @param variantsData
-   * @param context - the price selection context to use
-   * @return A list of products with variants decorated with prices
-   */
-  async setVariantPrices(
-    variantsData: { variant: ProductVariant; quantity?: number }[],
-    context?: PriceSelectionContext
-  ): Promise<PricedVariant[]>
-
   /**
    * Set additional prices on a list of product variants.
    * @param variantsData
    * @param context - the price selection context to use
    * @return A list of products with variants decorated with prices
    */
   async setVariantPrices(
-    variantsData:
-      | ProductVariant[]
-      | { variant: ProductVariant; quantity?: number }[],
+    variantsData: ProductVariant[],
```

Comment:
nit: 
```suggestion
    variants: ProductVariant[],
```

Reply:
@adrien2p this change should be reflected in the argument usage in the method body